scripts/1-hyperkube-release.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -euo pipefail
+
+echo "RELEASE_TAG=${RELEASE_TAG}"
+echo "PATCHES_FROM=${PATCHES_FROM}"
+echo "DRY_RUN=${DRY_RUN}"
+
+RELEASE_BRANCH="coreos-hyperkube-${RELEASE_TAG}"
+PATCHSET_BRANCH="${RELEASE_TAG}-patchset"
+
+git config user.name "jenkins-kube-lifecycle"
+git config user.email "jenkins-kube-lifecycle@example.com"
+git remote add coreos git@github.com:coreos/kubernetes.git
+
+# Create a release branch from vanilla upstream release tag
+RB_EXISTS=$(git ls-remote coreos ${RELEASE_BRANCH})
+if [ -n "${RB_EXISTS}" ]; then
+    echo "Release branch ${RELEASE_BRANCH} already exists. Skipping"
+else
+    echo "Creating release branch: ${RELEASE_BRANCH}"
+    git checkout ${RELEASE_TAG} -b ${RELEASE_BRANCH}
+    if [ "${DRYRUN}" = false ]; then
+        git push coreos ${RELEASE_BRANCH}
+    fi
+fi
+   
+
+# Create a branch containing previous release patchset 
+# TODO(pb) commented out opening a pull request with hub command until credentials are setup
```

Comment:
Do we want to do this? This would be another reason to use the "personal access token" instead of the SSH key as well...

Reply:
A personal access token is tied to a user, and gets access to their account. We are not using a process account (currently), but a [deploy key](https://github.com/coreos/kubernetes/settings/keys). The deploy key gets access to a single project and isn't tied to a user. Do we have a process github account already?

That comment is probably dead as well, because the credentials are setup at that point.

Reply:
We use a PAT for coreosbot (thats how the pull request builder can comment). But yeah, deploy keys sound like they have a much smaller blast radius. Guess it only matters if we want the PR opening automated.